client/app/pages/alert/components/Criteria.jsx

```diff
@@ -1,22 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { head, includes, toString, map } from 'lodash';
+import { head, includes, toString } from 'lodash';
 
 import Input from 'antd/lib/input';
 import Icon from 'antd/lib/icon';
 import Select from 'antd/lib/select';
+import Divider from 'antd/lib/divider';
 
 import { AlertOptions as AlertOptionsType } from '@/components/proptypes';
 
 import './Criteria.less';
 
 const CONDITIONS = {
-  'greater than': '>',
-  'less than': '<',
-  equals: '=',
+  '>': '\u003e',
+  '>=': '\u2265',
+  '<': '\u003c',
+  '<=': '\u2264',
+  '==': '\u003d',
+  '!=': '\u2260',
```

Comment:
I don't actually know and didn't wanna take the risk of a char rendering issue.
If you know this to be a non-issue, let's swap it for actual chars.

Reply:
I think it shouldn't be an issue for everyone who use proper IDEs or modern text editors. But it may be an issue for ones who will try, say, to print this file in Windows command line (not sure who may want to do this though)

Reply:
I will give this a try next time I boot Windows :)